Abcrete/BangazonAPI
# deleted commented code on CustomerController.cs, Customer.cs, Bangazo…

…nContext.cs. Changed Customer.LastLogin to manually entered rather than database generated

## Status

Choose:
**READY**

## Migrations
YES

## Description
I removed commented out code to clean up Customer.cs, CustomerController.cs, and BanganzonContext.cs. Additionally, I removed the "Required" and "DatabaseGererated" provisions from Customer.LastLogin 


## Todos
You'll need to reinitialize your database file to test the changes. Check that a customer can be added without the "LastLogin" property populated. Next, test manual entry of the "LastLogin" property using the "YYYY-MM-DDTHH:MM:SS" format.


## Steps to Test or Reproduce
1. git checkout master
2. git fetch 
3. git checkout jordan-done
4. delete your migrations directory and database file
5. in terminal, type "dotnet ef migrations add Initial"
6. Check the initial file in migrations directory to verify that "LastLogin" column on "Customer" table builder is set to "nullable: true"
7. in terminal, type "dotnet ef database update"
8. Use Postman or Terminal to post a new customer

## Impacted Areas in Application
Customer.cs
CustomerController.cs
Migrations
your local database file

## Link to Feature Ticket
#65
